src/modules/m_cgiirc.cpp

```diff
@@ -81,6 +82,14 @@ class CommandWebirc : public Command
 			if(user->registered == REG_ALL)
 				return CMD_FAILURE;
 
+			irc::sockets::sockaddrs ipaddr;
+			if (!irc::sockets::aptosa(parameters[3], 0, ipaddr))
+			{
+				IS_LOCAL(user)->CommandFloodPenalty += 5000;
```

Comment:
This is not harmful as the command is local-only. We should probably migrate to SplitCommand but I didn't want to make such an invasive change in 2.0.